docs/insights/menu-button.model.md

```diff
@@ -0,0 +1,54 @@
+---
+Added: v2.0.0
```

Comment:
I don't think we need a separate md file for this model. even paste the code in the readme is not a good practice can go easily out of sync